runtime/components/jacMachine/include/features/nodeModules.hpp

```diff
@@ -148,6 +154,284 @@ class NodeModuleLoader {
         return path;
     }
 
+    // Functions below are copied from duktape_v2.6.0-src/extras/module-node/duk_module_node.c
+    // They are static there, so we can't just use them.
```

Comment:
Stálo by za to ještě kompletně tím pádem z projektu zahodit závislost na `duk_module_node.c`, jelikož ho celý reimplementujeme.